shared/actions/teams.tsx

```diff
@@ -432,11 +432,14 @@ function* removeMember(_: TypedState, action: TeamsGen.RemoveMemberPayload, logg
   try {
     yield RPCTypes.teamsTeamRemoveMemberRpcPromise(
       {
-        allowInaction: true,
-        email: '',
-        inviteID: '',
+        member: {
+          assertion: {
```

Comment:
do you really need to call this field "assertion"?

Reply:
You can also pass it things like alice@twitter, etc. You could before too, now it's just clearer